app/views/users/_create_form.html.erb

```diff
@@ -105,24 +108,25 @@
 
     <div class="col-md-12 form-group form-inline" style="clear:both;padding-top:10px;">
         <!-- button for creating new users -->
-        <button style="margin-right:8px;" class="btn btn-lg btn-primary btn-save" type="submit" tabindex="7"><%= t('users._form.sign_up') %></button>
+        <button style="margin-right:8px;" class="btn btn-lg btn-primary btn-save sranje" type="submit" tabindex="7"><%= t('users._form.sign_up') %></button>
         <span>or <a id="toLogin" href="/login">
           <%= t('users._form.log_in') %>
         </a>
         </span>
 
+      <!--Validation library-->
       <script>
-        (function() {
-          $('.btn-save').click(function onClick(e) {
-            $(this).addClass("disabled") // disable the button after it is clicked
-                   .html("<i class='fa fa-spinner fa-spin'></i>"); // make a spinner that spins when clicked
-          })
-        })();
+        // (function() {
```

Comment:
Why you commented it out?

Reply:
I commented it out because I handle this button enable/disable state.

Reply:
I commented it out because I handle this button enable/disable state.

Reply:
I fixed it, it's required for the spinner. Thanks for pointing that out.